### ISSUE TITLE
Login using Bitbucket failed

### ISSUE THREAD

**Issue by lordlaw** [gitscrum-core/laravel-gitscrum#326]
## Detailed description

BUG ?
i had configure the bitbucket for login,
when i click on bitbucket grant access, it redirect to error page

Laravel \ Socialite \ Two \ InvalidStateException
No message
----------------------------------------------------------
Laravel\Socialite\Two\InvalidStateException…\vendor\laravel\socialite\src\Two\AbstractProvider.php 209

## Your environment
PHP 7.1.10
Apache2.4
Windows

at localhost

**Comment by rylxes** [comment]
i just tried it now, ... its working fine, maybe the configuration you set is not correct

**Comment by AgusRdz** [comment]
Same here, isn't replicable.

Try check your consumer permissions and then run
```
php artisan config:clear
php artisan config:cache
```

**Comment by miekassu** [comment]
I'm having exactly same issue with Github login. Yesterday I setup everything from scratch and it was fine and now I'm getting the same error.

I tried:
php artisan config:clear = no effect
php artisan config:cache = no effect
deleting cookies = redirected me to github login page, after login same error occurred

enviroment:
Ubuntu 17.10
PHP 7.1.17
Apache/2.4.27
Hosten on Azure

**Comment by frnc96** [comment]
I have the same issue when logging in with bitbucket. I deployed gitscrum in my own Digital Ocean droplet and access it through it's own subdomain (gitscrum.frenc.cc). When i log in on desktop i get the Undefined index Exception but when i log in through my mobile browser i get the same exception as @lordlaw . What should i do to get this app running?

**Comment by usman244718** [comment]
> I have the same issue when logging in with bitbucket. I deployed gitscrum in my own Digital Ocean droplet and access it through it's own subdomain (gitscrum.frenc.cc). When i log in on desktop i get the Undefined index Exception but when i log in through my mobile browser i get the same exception as @lordlaw . What should i do to get this app running?

Yes, I am available. Please share some details about the task. I am eager to collaborate with you. with 6+ years experience, please dm me if you want my portfolio and resume. #fullstack development, #graphic designing
